modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -24,11 +24,23 @@ interface RenderedDialog<T extends Dialog.DialogData> {
   readonly instanceApi: Dialog.DialogInstanceApi<T>;
 }
 
+const getInlineDialogSizeClasses = (size: Dialog.DialogSize): string[] => {
+  switch (size) {
+    case 'large':
+      return [ 'tox-dialog--width-lg--inline' ];
+    case 'medium':
+      return [ 'tox-dialog--width-md' ];
```

Comment:
This feels a bit weird to me that we use `tox-dialog--width-md` for the medium case which has no reference to inline in the class name but then have `inline` in the class name for the large case

Reply:
The original `large` CSS styling was not functioning properly with the inline dialog. The `medium` and `normal` CSS classes, on the other hand, were working as intended without any modifications, which is why they were reused in the CSS code. But yeah it does feels a bit weird. 

We could potentially breakdown the css classes in the future to have `tox-modal-dialog` and `tox-inline-dialog` for types of dialog with `dialog-width-md` and `dialog-width-lg` for different sizes